### ISSUE TITLE
Timing-based note colouring has little-to-no effect on skins with darker notes

### ISSUE THREAD

**Issue by Hydria** [ppy/osu#19554]
### Type

Cosmetic

### Bug description

Currently a user who wants to experience the timing based note colouring feature for mania is forced to use a note skin that has the notes be a lighter colour, as enabling it for a note skin that is dark or black has basically no effect on the note colour. 
This can be troublesome for users with no experience in skin editing or just prefers to use a darker 1/1 note.

The video below shows 0%, 100% and 10% brightness.

Skin can be downloaded here for testing: https://drive.google.com/drive/folders/1yAzwpCvzeMZCfPV4hXjiQooRvUWzIlEm

### Screenshots or videos

https://user-images.githubusercontent.com/13346079/182440858-30070b43-3cc4-45f4-90e5-ce48e61b6399.mp4

### Version

2022.731.1-lazer

### Logs

[performance-draw.log](https://github.com/ppy/osu/files/9244561/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9244562/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9244563/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9244564/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9244569/updater.log)
[database.log](https://github.com/ppy/osu/files/9244571/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9244572/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9244573/network.log)
[performance.log](https://github.com/ppy/osu/files/9244574/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9244575/performance-audio.log)

**Comment by mk56-spn** [comment]
i dont think this is solveable with old skins since unlike normal osu! mania notes to my knowledge lack an accent colour, so it just tries to tint the base colour, and well, attempting to tint black in such a way still results in black
